specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -2267,6 +2271,26 @@
         }
       }
     },
+    "Sku": {
+      "description": "Sku of Azure Spring Cloud",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the Sku",
+          "type": "string"
+        },
+        "tier": {
+          "description": "Tier of the Sku",
+          "type": "string"
+        },
+        "capacity": {
+          "format": "int32",
+          "description": "Current capacity of the target resource",
+          "type": "integer",
+          "readOnly": true
+        }
```

Comment:
what does this represent if this is readonly?  Typically this is used for scale out/in

Reply:
fixed